hq/app/views/documentation/unknown.scala.html

```diff
@@ -0,0 +1,9 @@
+@(file: String)(implicit assets: AssetsFinder)
+
+<h2>Not found</h2>
+<p>No documentation found on "@file".</p>
+<p>Consider searching on
+    <a target="_blank" href="https://www.google.com/search?q=@file">Google</a>, or
```

Comment:
I know that we have used target blank elsewhere, but I would prefer avoiding it as far as possible.

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Both of these would be okay opening in same window